exercises/luhn/canonical-data.json

```diff
@@ -1,7 +1,12 @@
 {
   "valid": [
     {
-      "description": "single digit strings can not be valid",
+      "description": "zero length strings are not valid",
+      "input": "",
+      "expected": false
```

Comment:
Probably not, the single character test cases should cover this. I'm happy to remove this commit unless anyone else can think of an example...